lib/pbench/server/api/__init__.py

```diff
@@ -23,7 +23,9 @@
 from pbench.server.api.resources.query_apis.datasets_list import DatasetsList
 from pbench.server.api.resources.query_apis.datasets_publish import DatasetsPublish
 from pbench.server.api.resources.query_apis.elasticsearch_api import Elasticsearch
-from pbench.server.api.resources.query_apis.index_mappings import IndexMappings
+from pbench.server.api.resources.query_apis.metadata_index.index_mappings import (
+    IndexMappings,
+)
```

Comment:
Ugh; our nested paths are getting too long when `black` won't even let us import a single name from a module!

Not really a serious comment; and yet, `metadata_index` is relatively long and not all that meaningful... nor, for that matter, is `query_apis`. Sigh ... but let's not worry about it for this PR. (Having brought it up out of nowhere...)

Reply:
Perhaps we don't need "index" in _both_ `metadata_index` **and** `index_mappings`?

Reply:
Webb, I think Metadata is a very broad term in our server context and it can mean multiple things, like metadata_table/user_metadata/datasets_metadata etc but I also concur that `metadata_index` is also not very useful name